service/db.go

```diff
@@ -190,14 +190,32 @@ func insertToken(uuid string, token string) error {
 	return nil
 }
 
+func testUserExist(uuid string) (bool, error) {
+	if uuid == "" {
+		return false, errInvalidUUID
+	}
+
+	command := `SELECT uuid FROM user_svc.accounts WHERE EXISTS (SELECT 1 from user_svc.accounts where uuid = $1)`
+	result, err := postgresDB.Query(command, uuid)
+
+	if err != nil {
+		return false, err
+	}
+
+	exists := result.Next()
```

Comment:
We should check what we got.

Example: https://astaxie.gitbooks.io/build-web-application-with-golang/en/05.4.html
https://golang.org/pkg/database/sql/#Rows
```
for rows.Next() {
            var uuid string
            var firstName string
            var lastName string
            err = rows.Scan(&uid, &firstName, &lastName)
           //  checkErr(err)
           // same(uuid with given uuid in the parameter)?
        }
```